### PR TITLE
Renamed various config values to be backwards compatible

### PR DESCRIPTION
**Pull request by A-UNDERSCORE-D**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -341,7 +341,7 @@ def __init__(self, do_winsparkle=True) -> None:
         self.identifier = applongname
         if (outdir_str := self.get_str('outdir')) is None or not pathlib.Path(outdir_str).is_dir():
             docs = known_folder_path(FOLDERID_Documents)
-            self.set('outdir',  docs if docs is not None else str(self.home))
+            self.set('outdir',  docs if docs is not None else str(self.home_path))
```
Comment (Athanasius) [inline config.py:344]: This could just be `self.home` now, no `str(...)`, yes ?

File: config.py [hunk 2]
```diff
@@ -702,7 +702,7 @@ def __init__(self, filename: Optional[str] = None) -> None:
             self.config.add_section(self.SECTION)
 
         if (outdir := self.get_str('outdir')) is None or not pathlib.Path(outdir).is_dir():
-            self.set('outdir', str(self.home))
+            self.set('outdir', str(self.home_path))
```
Comment (Athanasius) [inline config.py:705]: Again just `self.home`

**Comment by Athanasius** [comment]
Just those two `self.home` bits then.

**Comment by A-UNDERSCORE-D** [comment]
Whoops.
Both of those resolved

**Pull request merged**